curiosrc/web/static/pipeline_porep.html

```diff
@@ -2,11 +2,13 @@
 <head>
     <title>Lotus Provider PoRep Pipeline</title>
     <script src="https://unpkg.com/htmx.org@1.9.5" integrity="sha384-xcuj3WpfgjlKF+FXhSQFQ0ZNr39ln+hwjN3npfM9VBnUskLolQAcN80McRIVOPuO" crossorigin="anonymous"></script>
+    <script type="module" src="/ux/curio-ux.mjs"></script>
     <script type="module" src="chain-connectivity.js"></script>
     <link rel="stylesheet" href="main.css">
     <link rel='stylesheet' href='https://cdn.jsdelivr.net/npm/hack-font@3.3.0/build/web/hack-subset.css'>
 </head>
-<body>
+<body style="visibility: hidden">
+<curio-ux>
```

Comment:
This wrapper seems to break the htmx `hx-get` below, which breaks this page (pipeline state tables don't render at all now)

![2024-04-19-095834_3837x912_scrot](https://github.com/filecoin-project/lotus/assets/3867941/9cb55d9c-04a3-4fdd-b659-ca40ec4078a6)

Reply:
For reference before the changes here it looked like this
![2024-04-19-100542_1564x1342_scrot](https://github.com/filecoin-project/lotus/assets/3867941/70b20f50-c97a-44b2-835c-8e010505ca2f)